src/main/java/frc/robot/subsystems/color_wheel/commands/FindColor.java

```diff
@@ -0,0 +1,92 @@
+package frc.robot.subsystems.color_wheel.commands;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.Constants;
+import frc.robot.subsystems.color_wheel.ColorWheel;
+
+/**
+ * Spin the wheel until you reach the desired color.
+ */
+public class FindColor extends CommandBase {
+    private ColorWheel colorWheel;
+    private String color;
+    private double power;

```

Comment:
```suggestion
    private final ColorWheel colorWheel;
    private final String color;
    private final double power;

```
(I only added this because in Spin command you made it final...)